server/documents/elements/label.html.eco

```diff
@@ -333,16 +373,90 @@ themes      : ['Default', 'GitHub']
   <div class="example" data-use-content="true" data-class="icon">
     <h4 class="ui header">Icon</h4>
     <p>A label can include an icon</p>
-    <a class="ui label">
+    <div class="ui label">
       <i class="mail icon"></i>
       Mail
-    </a>
-    <a class="ui label">
-      Tag
-      <i class="delete icon"></i>
-    </a>
+    </div>
+    <div class="ui label">
+      <i class="checkmark icon"></i>
+      Test Passed
+    </div>
+    <div class="ui label">
+      <i class="dog icon"></i>
+      Dog
+    </div>
+    <div class="ui label">
+      <i class="cat icon"></i>
+      Cat
+    </div>
+  </div>
+  
+  <div class="another example" data-use-content="true">
+    <div class="ignored ui info message">Icons can be placed in the right side of label.</div>
+
+    <div class="ui right icon label">
+      Mail
+      <i class="mail icon"></i>
+    </div>
+    <div class="ui right icon label">
+      Test Passed
+      <i class="checkmark icon"></i>
+    </div>
+    <div class="ui right icon label">
+      Dog
+      <i class="dog icon"></i>
+    </div>
+    <div class="ui right icon label">
+      Cat
+      <i class="cat icon"></i>
+    </div>
+  </div>
+
+  <div class="another example">
+    <div class="ignored ui info message">Label can contains icon without text.</div>
+
+    <div class="ui icon label">
+      <i class="mail icon"></i>
+    </div>
+    <div class="ui icon label">
+      <i class="dog icon"></i>
+    </div>
+    <div class="ui icon label">
+      <i class="cat icon"></i>
+    </div>
   </div>
 
+  <div class="another example" data-use-content="true">
+    <div class="ignored ui warning message">
+      For backward-compatibility during v2.x, <code>close</code>/<code>delete</code> icon...
+      
+      <ul>
+        <li>requires <code>left icon</code> added to parent <code>label</code> when the icon is placed in the left of label.</li>
+        <li>do not require <code>right icon</code> added to parent <code>label</code> when the icon is placed in the right of label.</li>
+      </ul>
+      
+      This rule may be simplified in a later minor or major version".
```

Comment:
Whats the point of the `"` at the end?

Reply:
Oops.

Reply:
Fixed in https://github.com/fomantic/Fomantic-UI-Docs/pull/72/commits/56d1d1730bb8970be59d43816802f0bf4961b36b

Reply:
Fixed in https://github.com/fomantic/Fomantic-UI-Docs/pull/72/commits/56d1d1730bb8970be59d43816802f0bf4961b36b